profiles/base/package.use.stable.mask

```diff
@@ -4,6 +4,10 @@
 # This file requires eapi 5 or later. New entries go on top.
 # Please use the same syntax as in package.use.mask
 
+# Zen <dev@pyl.onl> (2024-06-29)
+# All versions of UGRD on all architectures are currently unstable.
+sys-kernel/installkernel ugrd
+
```

Comment:
```suggestion
```

This one is no longer required (whole commit 0a42a8aff95ac4a9d15e3fe8bef5337fb57ae8d1 can be dropped, it's superseded by commit 9a3942333cd59af1c5c9658c19c6a31f1ccdf05b)